matrix-neoboard-widget/index.html

```diff
@@ -27,6 +27,38 @@
       window.process = window.process ?? {};
       window.process.env = import.meta.env;
     </script>
+
+    <!-- Define font faces -->
+    <style>
```

Comment:
The font definition should live inside React SDK, so that it works in NeoBoard standalone as well. See https://mui.com/material-ui/customization/typography/ for example

Reply:
Font faces are now defined in the theme